src/core/moves/MetropolisHastingsMove.cpp

```diff
@@ -370,36 +370,38 @@ void MetropolisHastingsMove::performMcmcMove( double prHeat, double lHeat, doubl
     for (auto node: touched_nodes)
         node->touch();
 
+    bool fail_probability = not std::isfinite(ln_hastings_ratio);
+
```

Comment:
It's not clear to me why `Rb::isAComputableNumber` has been replaced by `std::isfinite` for the Hastings ratio but not the likelihood or prior ratios

Reply:
Never mind, this was done by a subsequent commit